.storybook-s2/docs/Icons.jsx

```diff
@@ -0,0 +1,35 @@
+import icons from '@react-spectrum/s2/s2wf-icons/*.svg';
+import { style } from '../../packages/@react-spectrum/s2/style/spectrum-theme' with {type: 'macro'};
+import {ActionButton, Text} from '@react-spectrum/s2';
+import {P, Code, Pre} from './typography';
+import {highlight} from './highlight' with {type: 'macro'};
+
+export function Icons() {
```

Comment:
Nice! I started working on a grid and filtering version of this because I needed to see all the icons to migrate an app. Thanks!